python/example_code/lookoutvision/.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
+        "."
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
Don't include IDE settings in the repo--you can add .vscode to .gitignore if you like.

Reply:
Removed.